config-reloader/templates/fluent.conf

```diff
@@ -1,7 +1,7 @@
 # dont modify this file when building on top of the image
 
 <system>
-  log_level info
+  log_level {{ .FluentdLogLevel }}
```

Comment:
This is correct as it injects it using the golang variables and the way it is named here is correct. 👍